app/src/main/java/dev/pinkroom/marketsight/ui/theme/Theme.kt

```diff
@@ -16,40 +16,33 @@ import androidx.compose.ui.platform.LocalView
 import androidx.core.view.WindowCompat
 
 private val DarkColorScheme = darkColorScheme(
-    primary = Purple80,
-    secondary = PurpleGrey80,
-    tertiary = Pink80
+    primary = BlackBackground,
+    onPrimary = White,
+    surfaceVariant = Gray,
+    surface = BlackBackground,
+    onSurfaceVariant = White,
 )
 
 private val LightColorScheme = lightColorScheme(
-    primary = Purple40,
-    secondary = PurpleGrey40,
-    tertiary = Pink40
-
-    /* Other default colors to override
-    background = Color(0xFFFFFBFE),
-    surface = Color(0xFFFFFBFE),
-    onPrimary = Color.White,
-    onSecondary = Color.White,
-    onTertiary = Color.White,
-    onBackground = Color(0xFF1C1B1F),
-    onSurface = Color(0xFF1C1B1F),
-    */
+    primary = GrayBackground,
+    onPrimary = Black,
+    surfaceVariant = White,
+    surface = GrayBackground,
+    onSurfaceVariant = Black,
 )
 
 @Composable
 fun MarketSightTheme(
     darkTheme: Boolean = isSystemInDarkTheme(),
     // Dynamic color is available on Android 12+
-    dynamicColor: Boolean = true,
+    dynamicColor: Boolean = false,
     content: @Composable () -> Unit
```

Comment:
I know that this code wasn't touched but always try to use a trailing comma as defined in Kotlin conventions: https://kotlinlang.org/docs/coding-conventions.html#trailing-commas

Reply:
Thanks, I will change